packages/core/useScroll/index.ts

```diff
@@ -137,6 +145,40 @@ export function useScroll(
       throttle ? useThrottleFn(onScrollHandler, throttle) : onScrollHandler,
       eventListenerOptions,
     )
+
+    useEventListener(element, 'wheel', (e: WheelEvent & {
+      wheelDelta: number
+      wheelDeltaX: number
+      wheelDeltaY: number
+    }) => {
+      const direction = unref(wheelDirection) ?? 'auto'
+
+      if (direction === 'auto')
+        // default wheel event
+        return
+
+      // Take over wheel events
+      e.preventDefault()
+
+      const ele = unref(element) as HTMLElement
+
+      if (direction === 'horizontal') {
+        let wheelDelta: number = Math.abs(e.wheelDeltaX) >= Math.abs(e.wheelDeltaY) ? e.wheelDeltaX : e.wheelDeltaY
+        wheelDelta = -(isWindowsOS ? wheelDelta / 3 : wheelDelta)
+
+        ele.scrollTo({
+          left: ele.scrollLeft + wheelDelta,
+        })
+      }
+      else if (direction === 'vertical') {
+        let wheelDelta: number = Math.abs(e.wheelDeltaY) >= Math.abs(e.wheelDeltaX) ? e.wheelDeltaY : e.wheelDeltaX
+        wheelDelta = -(isWindowsOS ? wheelDelta / 3 : wheelDelta)
```

Comment:
Why? Can you add comments with some references?

Reply:
It can be removed. At present, the scrolling distance under windows is a bit too far. Maybe can add options to control the ratio later as needed.